Userland/Applications/Piano/MainWidget.cpp

```diff
@@ -100,6 +105,20 @@ ErrorOr<void> MainWidget::add_track_actions(GUI::Menu& menu)
     return {};
 }
 
+void MainWidget::update_selected_track()
+{
+    if (static_cast<size_t>(m_track_manager.track_count()) > m_track_controls.size())
```

Comment:
`TrackManager::track_count()` should be made a `size_t` since that is its internal return value as well. Also allows you to get rid of a C-style cast in `TrackManager::set_current_track`.